backend/.env.prod

```
@@ -0,0 +1,106 @@
+# On prod we use AWS IAM instead of static credentials. Therefore, we don't need to set
+# it on prod.
+# AWS_ACCESS_KEY_ID=dummy
+# AWS_SECRET_ACCESS_KEY=dummy
+
+
+########################################################
+# Server
+########################################################
+SERVER_ENVIRONMENT=local
+SERVER_SIGNING_KEY=SErq6tYWOXsCQZ0B-ynjAIOxVFyOQX71E8vprZx6Msg
+SERVER_JWT_ALGORITHM=HS256
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVER_JWT_ALGORITHM key should go before the SERVER_SIGNING_KEY key

(UnorderedKey)